cypress/integration/pages/storyPage/tests.js

```diff
@@ -45,7 +47,15 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
         }
       });
     });
-
+    it('FOR /news/uk-60508671.amp ONLY - should render topic tags if they are in the json, and they should navigate to correct topic page', () => {
+      if (service === 'news' && Cypress.env('APP_ENV') !== 'local') {
+        const url = '/news/uk-60508671.amp?renderer_env=live';
```

Comment:
This will visit https://www.bbc.com/news/uk-60508671.amp?renderer_env=live when running the live env, but it should be fine

Reply:
Ha yes true. Odd, but harmless.